infra/base-images/base-builder-new/install_swift.sh

```diff
@@ -0,0 +1,54 @@
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+#!/bin/bash
+
+# generic swift
+apt-get update && apt install -y wget \
```

Comment:
I'm confused why we are installing all of these packages now and then other packages later, but I'm guessing this code existed before your PR so it isn't your fault.

Reply:
Agreed I should just group them, since it's not like there's any layers being cached here.

Reply:
I think it is still good to have 2 apt commands : one is for the swift compiler, the other one for the symbolizer (which hopefully will no longer be a custom patch to build)
But we can have them one right after the other one

Reply:
I separated out the packages meant for each for clarity.